README.md

```diff
@@ -419,6 +419,50 @@ On Linux, it is often necessary to first update `pip` to a version that supports
 
 ### pip installation: GPU (CUDA)
 
+<!---
+
+JAX provides pre-built CUDA wheels for **Linux only**. Installing on other
+operating systems is possible, but requires [building from
+source](https://jax.readthedocs.io/en/latest/developer.html#building-from-source).
+
+There are two ways to install JAX with CUDA support, either by installing CUDA
+using `pip` or by using a self-installed CUDA.
+
+#### Using a `pip`-installed copy of CUDA and CuDNN (recommended)
+
+NVIDIA provides almost everything JAX needs as wheels on PyPi with the exception
+of the GPU drivers. Installing CUDA from PyPi is the easiest way to install JAX
+and we strongly recommend it. The main downside is that NVIDIA's CUDA wheels are
+large (approximately 2GB), and they may duplicate an existing copy of CUDA on
+your system. If you would prefer to use a self-installed copy of CUDA see the
+next section. However we strongly recommend using a `pip` installed copy because
+it is simpler and more likely to work.
+
+First [install NVIDIA's GPU drivers](https://www.nvidia.com/download/index.aspx)
+if you have not already done so. The minimum version required by JAX is
+[450.80.02](https://docs.nvidia.com/deploy/cuda-compatibility/index.html#default-to-minor-version),
+however we recommend you install the latest driver for your platform. Some
+functionality (notably parallel compilation) will not work with older drivers.
+
+If you cannot update your NVIDIA driver easily and you have a datacenter
+or workstation GPU, you may be able to install and use the
+[CUDA forward compatibility packages](https://docs.nvidia.com/deploy/cuda-compatibility/)
+that NVIDIA provides for this purpose.
+
+
+To install JAX with CUDA support, run:
+```bash
+pip install --upgrade pip
+# Installs jax, jaxlib, CUDA, and CUDNN.
+# Note: wheels are only available on linux.
+pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
```

Comment:
I suppose this doc section is hidden for now.
This line is exactly the same as in the manual install path.
How do you plan to support both?
By creating a `jax[cuda_manual]` installation target?

Reply:
Yes. I didn't update the docs yet because we need to make a release, but if you look at the `setup.py` change below it adds an `existing_cuda_installation` target.

Reply:
I missed that. That looks great. Thanks.